compose/material3/benchmark/src/androidTest/java/androidx/compose/material3/benchmark/BottomAppBarBenchmark.kt

```diff
@@ -22,7 +22,6 @@ import androidx.compose.material.icons.automirrored.filled.ArrowBack
 import androidx.compose.material.icons.automirrored.filled.ArrowForward
 import androidx.compose.material3.BottomAppBar
```

Comment:
Why 80d88dc1b0d96901b35ceb59aa93bd4c60ed7142 is chosen, not the latest 2b53934ac6b8a887e71aa458b4a58c0c7d402f22 with the merged jb-main? We might lose the changes if we don't use the merged jb-main commit.

If you use scripts, it should choose the correct merge base. If not, we have to choose it correctly manually.

Reply:
2b53934ac6b8a887e71aa458b4a58c0c7d402f22 was made after this PR for follow-up purposes

Reply:
Previosly it was a mistake indeed, but after #1721, it should work. The script created copy based on old jb-main (merged). Later manual re-base to new state of jb-main doesn't wipe anything

Reply:
> Later manual re-base to new state of jb-main doesn't wipe anything

Can we just follow the processs? Merge jb-main, copy, empty merge back. Without rebasing and follow-up merges.

Otherwise I'll need to think how the rebase and follow-up merge without an empty merge can mess the things up. I have a suspicion that it can

Reply:
> the process

The script can do that, though, because the branch is different, you need to rename `integration` branch in the script